nip47/notifications/nip47_notifier.go

```diff
@@ -179,8 +183,8 @@ func (notifier *Nip47Notifier) notifySubscriber(ctx context.Context, app *db.App
 		return
 	}
 	logger.Logger.WithFields(logrus.Fields{
-		"notification": notification,
-		"appId":        app.ID,
+		// "notification": notification,
+		"appId": app.ID,
 	}).Info("Published notification event")
```

Comment:
is this a useful log?

Reply:
Probably only for debugging. I changed it to a debug log so it'll be filtered out in production